docker/Makefile

```diff
@@ -0,0 +1,30 @@
+  
+default:  
+	echo 'Options are bootstrap, start, monitor, stop, restart, nuke, clean (cleans untagged/unnamed images)'
+
+start:  
+	export COMPOSE_PROJECT_NAME=myaegee && sudo docker-compose up -d
+  
+bootstrap: start monitor
```

Comment:
(to give detail: on the same line there are "targets", on the different lines there are custom scripts that a target has to execute)